
omicR. This package has the following fuctions: 1) Create fasta files. 2) Download genomes from NCBI.3) Create genome database for BLAST. 4) Run BLASTn and selects the best match per sequence. 5) Alternative step: Select the best match from a BLAST 

Windows. 

Introduction

omicR creates fasta files, downloads genomes from NCBI using the refseq number, creates databases to run BLAST+, runs BLAST+ and filters these results to obtain the best match per sequence. 
These scripts can be used to run BLAST alignment of short-read (DArTseq data) and long-read sequences (Illumina, PacBio… etc). You can use reference genomes from NCBI, genomes from your private collection, contigs, scaffolds or any other genetic sequence that you would like to use as reference. 



Requirements: BLAST+ latest version: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ 


Download the zip directory “omicR.zip“.  Unzip this directory and double click the executable file “omicR.exe” with the image of the green parrot. This will open a window where you can run the scripts. You do not need to install anything as everything is compiled into this file and you can start running your analysis. 



Tutorial to use this scripts (~20 min): https://youtu.be/pdMio2vj-FM 

